src/components/ResourceOverview/HospitalOverviewPage.tsx

```diff
@@ -0,0 +1,293 @@
+  });
+
+  const handleClickOptions = (event) => {
+    selectHospital(event.currentTarget.getAttribute('data-id'));
+    setAnchorEl(anchorEl ? null : event.currentTarget);
+  };
+
+  const history = useHistory();
+  const handleClickEdit = () => {
+    const hospitalId = selectedHospital;
+    history.replace(`/manage/hospitals/edit/${hospitalId}`);
+  };
+
+  const handleClickDelete = () => {
+    const hospitalId = selectedHospital;
+    deleteHospital({ variables: { id: hospitalId } });
+    setOpenModal(false);
+  };
+
+  const cells = hospitals.map((hospital: Hospital) => {
+    return (
+      <TableRow key={hospital.id}>
+        <TableCell classes={{ root: dRow.root }}>{hospital.name}</TableCell>
+        <TableCell classes={{ root: optionBtn.root }}>
+          <IconButton data-id={hospital.id} onClick={handleClickOptions}>
+            <MoreHorizIcon style={{ color: Colours.Black }} />
+          </IconButton>
+        </TableCell>
+      </TableRow>
+    );
+  });
+
+  return (
+    <div className={classes.wrapper}>
+      <Typography variant="h5">Hospital Overview</Typography>
+      <Typography variant="body2" classes={{ body2: paraStyle.body2 }}>
+        A list of all hospitals that can be added to an event.
+      </Typography>
+
+      <TableContainer>
+        <Table classes={{ root: table.root }}>
+          <TableHead>
+            <TableRow>
+              <TableCell classes={{ root: hRow.root }}>Hospital Name</TableCell>
+              <TableCell />
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {cells}
+            <Popper
+              id={id}
+              open={open}
+              popperOptions={{
+                modifiers: { offset: { enabled: true, offset: '-69.5,0' } },
+              }}
+              anchorEl={anchorEl}
+            >
+              <div>
+                <Table className={classes.tablePopper}>
+                  <TableBody>
+                    <TableRow
+                      hover
+                      classes={{ hover: optionBtn.menuHover }}
+                      onClick={handleClickEdit}
+                    >
+                      <TableCell classes={{ root: optionBtn.menuCell }}>
+                        <Typography variant="body2">Edit</Typography>
+                      </TableCell>
+                    </TableRow>
+                    <TableRow hover onClick={() => setOpenModal(true)}>
+                      <TableCell classes={{ root: optionBtn.menuDelete }}>
+                        <Typography variant="body2">Delete</Typography>
+                      </TableCell>
+                    </TableRow>
+                  </TableBody>
+                </Table>
+              </div>
+            </Popper>
+          </TableBody>
+        </Table>
+      </TableContainer>
+      <Dialog classes={{ paper: dialogStyle.paper }} open={openModal}>
+        <DialogTitle classes={{ root: dialogStyle.dialogTitle }}>
+          <Typography variant="h6">
```

Comment:
I think this is causing the warning in the console `Warning: validateDOMNesting(...): <h6> cannot appear as a child of <h2>.` Either don't use `DialogTitle` or see if you can change `DialogTitle` to use the "h6" variant.